samples/client/petstore/typescript-angular-v4/npm/api/pet.service.ts

```diff
@@ -274,7 +274,7 @@ export class PetService {
      */
     public deletePetWithHttpInfo(petId: number, apiKey?: string, extraHttpRequestParams?: any): Observable<Response> {
         const path = this.basePath + '/pet/${petId}'
-                    .replace('${' + 'petId' + '}', String(petId));
+                    .replace('${' + 'petId' + '}', encodeURIComponent(String(petId)));
```

Comment:
Do you know where these non-related changes in the generated output comes from? Have you pulled the latest changes before running your enhancements? I think this `encodeURI` thing was introduced in another PR and therefore should be already present?

Reply:
this comes from the fact that in https://github.com/swagger-api/swagger-codegen/pull/6525 I did run the `bin/windows/typescript-angular2-petstore-all.bat` script, but forgot to run the `bin/windows/typescript-angular-v4-with-npm.bat` script. This shows that the CI tests somehow did not check the angular v4 output samples. Maybe this needs to be fixed? @sebastianhaas will you do that?